### PR TITLE
Update PP inference benchmark script

### PR DESCRIPTION
**Pull request by plusbang**
## Description

- Update PP inference benchmark script
Just need to run `run-pipeline-parallel-arc.sh` and modify `NUM_GPUS` in it to specify gpu number.
- Fix `Assertion 'input.scalar_type() == weight.scalar_type()' failed` when setting `torch_dtype=torch.float16`

### 4. How to test?
- [ ] Unit test
- [x] Local test

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 1]
```diff
@@ -1849,8 +1817,8 @@ def run_pipeline_parallel_gpu(repo_id,
                 print(output[0])
                 torch.xpu.empty_cache()
                 if i >= warm_up:
-                    result[in_out].append([model.first_cost, model.rest_cost_mean, model.encoder_time,
-                                           actual_in_len, actual_out_len, load_time, model.peak_memory, fp16])
+                    result[in_out].append([model.first_token_time, model.rest_cost_mean, 0,
+                                           actual_in_len, actual_out_len, load_time, 0, fp16])
```
Comment (Oscilloscope98) [inline python/llm/dev/benchmark/all-in-one/run.py:1821]: The final `0` and `fp16` can be removed :)

**Review by Oscilloscope98 — APPROVED** [review]
We could also update the readme here? https://github.com/intel-analytics/ipex-llm/tree/main/python/llm/dev/benchmark/all-in-one#config

Others LGTM

**Comment by plusbang** [comment]
> We could also update the readme here? https://github.com/intel-analytics/ipex-llm/tree/main/python/llm/dev/benchmark/all-in-one#config


Have updated :)

**Review by rnwang04 — APPROVED** [review]
LGTM

**Pull request merged**